src/app-layout/utils/use-drawers.ts

```diff
@@ -63,8 +63,10 @@ function useRuntimeDrawers(
   const onLocalDrawerChangeStable = useStableCallback(onActiveDrawerChange);
   const onGlobalDrawersChangeStable = useStableCallback(onActiveGlobalDrawersChange);
 
-  const drawersWereOpenRef = useRef(false);
-  drawersWereOpenRef.current = drawersWereOpenRef.current || !!activeDrawerId || !!activeGlobalDrawersIds.length;
+  const localDrawerWasOpenRef = useRef(false);
+  localDrawerWasOpenRef.current = localDrawerWasOpenRef.current || !!activeDrawerId;
+  const activeGlobalDrawersIdsStable = useRef<Array<string>>([]);
```

Comment:
nit: `activeGlobalDrawersIdsStable` could be named as `activeGlobalDrawersIdsStableRef` postfix it with `Ref`.